nuc_morph_analysis/analyses/neighbor_of_X/run_all_example_worfklows.py

```diff
@@ -0,0 +1,6 @@
+#%%
```

Comment:
I dont want to make more work for you but it might make more sense to have each figure be its own function. That way you load the data once in this workflow and plot each example.

Reply:
done